flocker/node/agents/test/test_blockdevice.py

```diff
@@ -1473,6 +1499,8 @@ def test_attach_volume_validate_size(self):
             new_volume.blockdevice_id, attach_to=self.this_node,
         )
 
+        self.assertNotEqual(attached_volume, None)
```

Comment:
Why are these assertions necessary (as opposed to any other random assertion)? Won't returned `None` cause later assertions to fail?